src/main/java/pl/connectis/Spotify_API_CLI/authorization/ClientCredentialsAuthorization.java

```diff
@@ -0,0 +1,74 @@
+package pl.connectis.Spotify_API_CLI.authorization;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Base64;
+
+
+@Slf4j
+@Component("client_credentials")
+public class ClientCredentialsAuthorization implements AuthorizationStrategy {
+
+    private final ApplicationContext context;
+    private final RestTemplate restTemplate;
+    private final HttpHeaders httpHeaders;
+    @Value("${authorizationMethod}")
+    private String authorizationMethod;
+    @Value("${clientId}")
+    private String clientID;
+    @Value("${clientSecret}")
+    private String clientSecret;
+    @Value("${accessTokenUri}")
+    private String accessTokenUri;
```

Comment:
Opakuj to w klase z annotacja `@ConfigurationProperties`

Reply:
ok